sdk/storage/AzureStorageBlob/Source/DataStructures/StorageBlobClientOptions.swift

```diff
@@ -48,11 +48,22 @@ public struct StorageBlobClientOptions: AzureConfigurable {
     public init(
         apiVersion: StorageBlobClient.ApiVersion = .latest,
         logger: ClientLogger = ClientLoggers.default(tag: "StorageBlobClient"),
-        maxChunkSizeInBytes: Int = 4 * 1024 * 1024 - 1
+        maxChunkSizeInBytes: Int = 4 * 1024 * 1024 - 1,
+        downloadConnectivityPolicy: TransferNetworkPolicy? = nil,
+        uploadConnectivityPolicy: TransferNetworkPolicy? = nil
     ) {
         self.apiVersion = apiVersion.rawValue
         self.logger = logger
         self.maxChunkSizeInBytes = maxChunkSizeInBytes
+        URLSessionTransferManager.shared.uploadConnectivityPolicy = uploadConnectivityPolicy ?? TransferNetworkPolicy(
```

Comment:
This logic isn't quite right. If the customer initializes a `StorageBlobClientOptions`, the settings it contains shouldn't have any effect until it's used (at the earliest - when the client is created).

Reply:
Updated the logic so the transfer manager can reference the policy directly from the client object.